rocky/reports/report_types/findings_report/report.html

```diff
@@ -50,7 +50,11 @@ <h3>{% translate "Findings" %}</h3>
                                 <h4 class="heading-normal">{% translate "Description" %}</h4>
                                 <p>{{ info.finding_type.description }}</p>
                                 <h4 class="heading-normal">{% translate "Source" %}</h4>
-                                <a href="{{ info.finding_type.source }}">{{ info.finding_type.source }}</a>
+                                {% if info.finding_type.source %}
+                                    <a href="{{ info.finding_type.source }}">{{ info.finding_type.source }}</a>
+                                {% else %}
+                                    <p>{{ info.finding_type.source }}</p>
+                                {% endif %}
```

Comment:
Does it make sense to even print None or Source here at all if there is no source?

Reply:
If we decide not to do this, we should also remove the option to display None from all other headings (Description, Impact and Recommendation).